example-app/src/reducers/reducer.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { selectIsPending } from 'redux-pending-effects';
+import SagaTester from 'redux-saga-tester';
+
+import { store } from '../store';
+import {
+  getPatents,
+  getLibraryContent,
+  getAstronomyPictureData
+} from '../actions';
+import { astronomyPictureActionNames, clearStoreActionName } from '../constants/actionNames';
+import { nasaService } from '../services/NasaService';
+import { astronomyPictureWorker } from '../sagas/astronomyPictureSagas';
+import { rootReducerWrapper } from './rootReducer';
+
+describe('store testing on RPE selector', () => {
+  afterEach(() => {
+    store.dispatch({
+      type: clearStoreActionName
+    });
+  });
+
+  it('should test selectIsPending changing on getPatents fetch', async () => {

```

Comment:
it exactly tests smth, you should not repeat the semantic behavior.
I mean it's so clear what it does, better describe what kind of behavior you want to test in this test

for example:

`should change ...`